### DISCUSSION TITLE
Goto RA/DEC confusion

### DISCUSSION THREAD

**Discussion by RoscoeD** [Stellarium/stellarium#3470] [Q&A] [answered]
Trying to shoot a target that is not in my ASI Air database.  So I'm attempting to frame it in macOS Stellarium application so I can get the proper RA/DEC and camera rotation.  I'll then manually enter those into my ASI Air.  Biggest issue is that there is no object in Stellarium that is in the center of my desired FOV that I can click on, so I have to frame it manually and extract the coordinates.  Let the struggles commence...

(Note I have my scope parameters in Stellarium)

First I "mouse" the screen around to the location I want edit the rotation angle until I get the desired framing.  According to the app screen itself, the RA/DEC coordinates shown in the upper left hand corner of the camera FOV box is the cross in the center of the FOV.  Sounds good.  Over time however I saw that the center had moved because couldn't click on a target to "lock" it.  SO I ran a script I found on line core.moveToRaDec("22h03m00s", "+73d04m57s"); to take me back to where I wanted to go.  The RA/DEC reported on the screen is not the RA/DEC I scripted the Goto.  Where does that script command send the screen, because it wasn't the sensor FOV center (close, but far enough that the target was not framed correctly)?

Another issue was no matter what I clicked I couldn't stop the "time".  I finally found a web page that said use "k" on the keyboard...but there ought to be a means to do that on the screen with a mouse.

Bottom line, I never could get the coords (with confidence) so no clue what I'm going to shoot...pretty sure someone part of the target is going to fall outside the FOV.  Messing with all of this cost me me almost 2 hours of nice clear sky.

**Comment by alex-w** [comment, marked as answer]
You can put the custom markers in the sky and select it as an object (details via F1) or use equatorial mount to "freeze a time".

> Another issue was no matter what I clicked I couldn't stop the "time". I finally found a web page that said use "k" on the keyboard...but there ought to be a means to do that on the screen with a mouse.

Bottom bar has a section to manage time speed and direction (right side of bar).

**Comment by RoscoeD** [comment]
Bottom bar section did not appear to stop the movement.  The RA/DEC of the FOV cross would drift no matter what button I clicked (unless I locked the screen to an object.

I'll look into the custom markers.  Thanks

**Comment by RoscoeD** [comment]
Custom marker...cool.  How do I precisely place it at a specific RA/DEC?
